src/middlewared/middlewared/plugins/iscsi_/alua.py

```diff
@@ -611,3 +613,49 @@ async def wait_for_alua_settled(self, sleep_interval=1, retries=10):
             await asyncio.sleep(sleep_interval)
             retries -= 1
         self.logger.warning('Gave up waiting for ALUA to settle')
+
+    @job(lock='force_close_sessions', transient=True, lock_queue_size=1)
+    async def force_close_sessions(self, job):
+        job.set_progress(0, 'Start force-close of iSCSI sessions')
+        self.logger.debug('Start force-close of iSCSI sessions')
+
+        await run('scst_util.sh', 'force-close')
```

Comment:
Is there a reason why we're fork'ing exec'ing in the main event loop to run some commands that we can do in native python?

Reply:
Could easily be writing to a **large** number of paths, and want to do it in parallel.

Reply:
Derp, I completely glanced over the trailing ampersand in the script :slightly_smiling_face: